ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/Errors.java

```diff
@@ -70,6 +72,13 @@ public static Response accessDenied(final String msg) {
         .build();
   }
 
+  public static Response forbiddenKafkaAccess(final Exception error) {
```

Comment:
There is a method in this class named `accessDenied`. Should we name this method similar to it to keep it consistent? Like `accessDeniedFromKaffka` or something?